src/main/java/oshi/hardware/platform/linux/LinuxSensors.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Oshi (https://github.com/dblock/oshi)
+ * 
+ * Copyright (c) 2010 - 2016 The Oshi Project Team
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ * dblock[at]dblock[dot]org
+ * alessandro[at]perucchi[dot]org
+ * widdis[at]gmail[dot]com
+ * https://github.com/dblock/oshi/graphs/contributors
+ */
+package oshi.hardware.platform.linux;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.json.Json;
+import javax.json.JsonArrayBuilder;
+import javax.json.JsonBuilderFactory;
+import javax.json.JsonObject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import oshi.hardware.Sensors;
+import oshi.util.FileUtil;
+
+public class LinuxSensors implements Sensors {
+    private static final Logger LOG = LoggerFactory.getLogger(LinuxSensors.class);
+
+    private JsonBuilderFactory jsonFactory = Json.createBuilderFactory(null);
+
+    private static final String HWMON = "/sys/class/hwmon/hwmon0/";
```

Comment:
The final number after hwmon indexes different sensors.  hwmon0 may be temp and hwmon1 may be fans, or they may be in a different order.  This needs rewriting to account for temp*_input and fan*_input being in any potential hwmon* path.